compliance/cis_k8s/rules/cis_5_1_3/rule.rego

```diff
@@ -0,0 +1,47 @@
+package compliance.cis_k8s.rules.cis_5_1_3
+
+import data.compliance.cis_k8s
+import data.compliance.lib.assert
+import data.compliance.lib.common
+import data.compliance.lib.data_adapter
+import future.keywords.in
+
+# Minimize wildcard use in Roles and ClusterRoles (Manual)
+
+# evaluate
+default rule_evaluation = false
+
+rule_evaluation {
```

Comment:
not sure what are you referring to, or how your suggestion differentiates logically from the original code (I guess it's not?), in any case, don't you think it will be less efficient?